### DISCUSSION TITLE
Adding a top level filter on collection for user ergonomics

### DISCUSSION THREAD

**Discussion by laekem34** [directus/directus#10549] [Feature Requests]
Hello,

It could be a great idea to be able to add "subfolders" under collection which are show a part of collection data pre-filtered
(See screens below for implementation example).

With large number of items, it could be more easier to have only items with a same common point. But often we have to swichter between  that "common point", which is not ergonomics when use the layout filter field: we add a filter to see only items we want, later we want another items so we erase and change the filter, and if we want to look first filtered items, we have erase and change the filter again.

With this feature, we could have a top level filter, like users page (with all user, and role filtered subfolders)
 
![1](https://user-images.githubusercontent.com/21008981/146380968-d25178fe-d302-4838-8e11-d5ca5aaf5494.png)
![2](https://user-images.githubusercontent.com/21008981/146380989-61996d27-30c2-4858-aa76-0816bd0f94eb.png)
![3](https://user-images.githubusercontent.com/21008981/146381011-f8670ffc-c1ff-4b50-b444-ed8dfe9c4880.png)
![4](https://user-images.githubusercontent.com/21008981/146381035-cd3c0b09-911c-4adf-aae0-3de4606ac6b9.png)

Thanks you !

**Comment by benhaynes** [comment]
I love the idea of customizable filters in the header, but before diving into that I want to be sure of one thing.

Have you seen our bookmarks? That seems to be exactly what you're describing. Creating a bookmark will store all of your filters, layout, options, etc... and is shown nested under the collection in the navigation. 🤔

**Reply by laekem34** [reply]
Haha sorry Benhaynes that's exactly that i had purpose ^^'
No need to add the same feature so.

Can i just purpose to add custom icon for bookmark, like the screen ?
Can i try to open a PR ?
![image](https://user-images.githubusercontent.com/21008981/146512880-d0974aeb-930e-494d-b71f-21834cd50200.png)

**Reply by laekem34** [reply]
Sometimes if i edit preset (and even if not edit), it is duplicated
And simple user have right to create its bookmark.
![image](https://user-images.githubusercontent.com/21008981/146534099-5476b3d5-74d4-44bf-9111-7a43b04118ad.png)

In my first approach, i would like create sub prefiltered collections and expose them to simple user which haven't right to edit them.
With presets, users have too control on them, even with minimal permissions.

**Reply by benhaynes** [reply]
Yeah, would love to get custom icons/colors in for bookmarks. We should also think through and clarify how Global/Role/User bookmarks are configured and managed.

**Reply by azrikahar** [reply]
This is currently being worked on over at #12031 👍

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
